packages/sui-test/bin/sui-test-e2e.js

```diff
@@ -37,19 +37,32 @@ program
     '-U, --userAgentAppend <userAgentAppend>',
     'Append string to UserAgent header.'
   )
+  .option(
+    '-UA, --userAgent <userAgent>',
+    'Overwrite string to UserAgent header.'
+  )
   .option('-s, --scope <spec>', 'Run tests specifying a subfolder of specs')
   .option('-G, --gui', 'Run the tests in GUI mode.')
   .on('--help', () => console.log(HELP_MESSAGE))
   .parse(process.argv)
 
-const {baseUrl, userAgentAppend, gui, screenshotsOnError, scope} = program
+const {
+  baseUrl,
+  userAgentAppend,
+  userAgent,
+  gui,
+  screenshotsOnError,
+  scope
+} = program
 const cypressConfig = {
   integrationFolder: path.join(TESTS_FOLDER, scope || ''),
   baseUrl,
   fixturesFolder: path.join(TESTS_FOLDER, 'fixtures')
 }
 
-if (userAgentAppend) {
+if (userAgent) {
+  cypressConfig.userAgent = `"${userAgent}"`
```

Comment:
This can be simplified with: `cypressConfig.userAgent = userAgent`

Reply:
You're right but, At the beginning I did it but I see that the function `objectToCommaString` sets the userAgent as `userAgent: 'undefined'` if you don't set --userAgend or --userAgentAppend because it will assign undefined in this property. So, I prefer not to modify `objectToCommaString`

Reply:
If cypress.json has userAgent: 'undefined', Cypress browser will has a 405 - Method Not Allowed (text/html) response

Reply:
Yes, sorry, I was wrong, the quotes are needed in this case ;) I had the same issue early this morning 😛